File: Entities/AutoNumberEn.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ServiceModel;
using System.Runtime.Serialization;

namespace HTS.SAS.Entities
{
    [System.SerializableAttribute()]
    //[DataContract]
    public class AutoNumberEn
    {
        private int ciSAAN_Code;
        private string csSAAN_Des;
        private string csSAAN_Prefix;
        private int ciSAAN_NoDigit;
        private int ciSAAN_StartNo;
        private int ciSAAN_CurNo;
        private string csSAAN_AutoNo;


        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public int SAAN_Code
        {
            get { return ciSAAN_Code; }
            set { ciSAAN_Code = value; }
        }


        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public string SAAN_Des
        {
            get { return csSAAN_Des; }
            set { csSAAN_Des = value; }
        }


        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public string SAAN_Prefix
        {
            get { return csSAAN_Prefix; }
            set { csSAAN_Prefix = value; }
        }


        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public int SAAN_NoDigit
        {
            get { return ciSAAN_NoDigit; }
            set { ciSAAN_NoDigit = value; }
        }


        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public int SAAN_StartNo
        {
            get { return ciSAAN_StartNo; }
            set { ciSAAN_StartNo = value; }
        }


        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public int SAAN_CurNo
        {
            get { return ciSAAN_CurNo; }
            set { ciSAAN_CurNo = value; }
        }


        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public string SAAN_AutoNo
        {
            get { return csSAAN_AutoNo; }
            set { csSAAN_AutoNo = value; }
        }

    }
}

File: BusinessEntities/AutoNumberEn.vb
Public Class AutoNumberEn

        Private _SAAN_Code As Integer
        Public Property SAAN_Code() As Integer
            Get
                Return _SAAN_Code
            End Get
            Set(ByVal value As Integer)
                _SAAN_Code = value
            End Set
        End Property

        Private _SAAN_Des As String
        Public Property SAAN_Des() As String
            Get
                Return _SAAN_Des
            End Get
            Set(ByVal value As String)
                _SAAN_Des = value
            End Set
        End Property

        Private _SAAN_Prefix As String
        Public Property SAAN_Prefix() As String
            Get
                Return _SAAN_Prefix
            End Get
            Set(ByVal value As String)
                _SAAN_Prefix = value
            End Set
        End Property

        Private _SAAN_NoDigit As Integer
        Public Property SAAN_NoDigit() As Integer
            Get
                Return _SAAN_NoDigit
            End Get
            Set(ByVal value As Integer)
                _SAAN_NoDigit = value
            End Set
        End Property

        Private _SAAN_StartNo As Integer
        Public Property SAAN_StartNo() As Integer
            Get
                Return _SAAN_StartNo
            End Get
            Set(ByVal value As Integer)
                _SAAN_StartNo = value
            End Set
        End Property

        Private _SAAN_CurNo As Integer
        Public Property SAAN_CurNo() As Integer
            Get
                Return _SAAN_CurNo
            End Get
            Set(ByVal value As Integer)
                _SAAN_CurNo = value
            End Set
        End Property

        Private _SAAN_AutoNo As String
        Public Property SAAN_AutoNo() As String
            Get
                Return _SAAN_AutoNo
            End Get
            Set(ByVal value As String)
                _SAAN_AutoNo = value
            End Set
        End Property

End Class
